scout/server/extensions/phenopacket_extension.py

```diff
@@ -101,10 +101,10 @@ def get_hash(self, phenopacket_hash):
 
         query = f"{self.url}/api/v1/phenopacket?hash={phenopacket_hash}"
 
-        try:
-            json_response = get_request_json(query)
-        except Exception as e:
-            LOG.warning(e)
+        json_response = get_request_json(query)
```

Comment:
Try/catch block is already present in [get_request_json function](https://github.com/Clinical-Genomics/scout/blob/a22110f94e40fd11f1f343559c8ff6e2afbe0979/scout/utils/scout_requests.py#L57)

Reply:
Right, good catch - in particular that function did not re-raise the exception like the main one did before.. 🙄